app/controllers/publish/courses/funding_type_controller.rb

```diff
@@ -92,6 +117,28 @@ def course_page_path
       def section_key
         'Funding type'
       end
+
+      def handle_redirect
+        if goto_visa_path?
+          redirect_to visa_sponsorship_path
+        elsif previous_tda_course_path?
+          redirect_to previously_defaulted_attributes
```

Comment:
I'd change to 
`redirect_to new_publish_provider_recruitment_cycle_courses_study_mode_path(previously_defaulted_attributes)`
because one thing is attributes another is the path

Reply:
Updated here: https://github.com/DFE-Digital/publish-teacher-training/pull/4223/commits/7ad5df9669bcf0a1c4d2df3f8e4e7917f78daeff